protocols/grpc/agent.proto

```diff
@@ -60,6 +60,7 @@ service AgentService {
 	rpc MemHotplugByProbe(MemHotplugByProbeRequest) returns (google.protobuf.Empty);
 	rpc SetGuestDateTime(SetGuestDateTimeRequest) returns (google.protobuf.Empty);
 	rpc CopyFile(CopyFileRequest) returns (google.protobuf.Empty);
+	rpc GetOOMEvents(GetOOMEventsRequests) returns (OOMEvents);
```

Comment:
The runtime/shim will poll this endpoint periodically to retrieve OOM events that have occurred since the last time it was polled. I haven't determined the poll period yet, but potentially something low like 500ms to ensure timely arrival of events.

Ideally this endpoint would be a stream endpoint, but ideally they shouldn't be used - explained here https://github.com/kata-containers/runtime/issues/2150#issuecomment-586346977

Reply:
Maybe you can use a sync call to wait until there is OOM events occur and then return the events, published the events to containerd and then another call again in a loop.

Reply:
I've updated the GRPC endpoint to return a single event at a time.